catroid/src/main/java/org/catrobat/catroid/ui/recyclerview/fragment/DataListFragment.kt

```diff
@@ -293,6 +292,34 @@ class DataListFragment : Fragment(),
         }
     }
 
+    private fun showEditDialog(selectedItems: List<UserData<*>>) {
+        val item = selectedItems[0]
+        val builder = TextInputDialog.Builder(requireContext())
+
+        builder.setHint(getString(R.string.data_value))
+            .setText(item.value.toString())
+            .setPositiveButton(
+                getString(R.string.ok)
```

Comment:
also please change the button title from "ok" to "save" as described in the ticket.

Reply:
Sure!